apps/finance/app/src/App.js

```diff
@@ -20,12 +20,11 @@ class App extends React.Component {
   static propTypes = {
     api: PropTypes.object,
     appState: PropTypes.object,
+    compactMode: PropTypes.bool,
+    isSyncing: PropTypes.bool,
   }
   static defaultProps = {
     isSyncing: true,
-    balances: [],
```

Comment:
Note to self: these aren't passed in anymore, as we now fetch the state from `api-react`'s `appState`.